package.json

```diff
@@ -2,22 +2,38 @@
   "name": "cs-applied-plan-portal",
   "version": "1.0.0",
   "description": "",
-  "main": "server.js",
+  "main": "index.js",
   "directories": {
     "doc": "docs"
   },
   "dependencies": {
+    "css-loader": "^3.4.2",
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
+    "extract-text-webpack-plugin": "^4.0.0-beta.0",
     "mysql": "^2.17.1",
-    "path": "^0.12.7"
+    "path": "^0.12.7",
+    "react": "^16.12.0",
+    "react-dom": "^16.12.0",
+    "style-loader": "^1.1.3"
   },
   "devDependencies": {
-    "eslint": "^6.8.0"
+    "@babel/core": "^7.0.0",
+    "@babel/preset-env": "^7.0.0",
+    "@babel/preset-react": "^7.0.0",
+    "babel-loader": "^8.0.0",
+    "eslint": "^6.8.0",
+    "html-webpack-plugin": "^3.2.0",
+    "http-server": "^0.12.1",
+    "webpack": "^4.41.5",
+    "webpack-cli": "^3.3.10",
+    "webpack-dev-server": "^3.10.1"
   },
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "node server.js"
+    "start": "node_modules/.bin/webpack-dev-server --mode development --open --hot",
```

Comment:
This will change the behavior of "npm start", but I don't see this as an issue as we will likely change it again soon as we merge front-end and back-end components.

Reply:
Yeah I figured we'd resolve this once we got together to discuss the merge